components/homeRouter.js

```diff
@@ -1,19 +1,43 @@
-import { createSwitchNavigator, createStackNavigator, createAppContainer } from 'react-navigation';
-
+import React from 'react';
+import { createSwitchNavigator, createStackNavigator, createAppContainer, createBottomTabNavigator } from 'react-navigation';
+import { Icon } from 'react-native-elements';
 //Import screens
 import LogInOptionsScreen from "../screens/LogInOptions";
 import CreateAccount from "../screens/CreateAccount";
 import StartScreen from "../screens/Start";
 import HomeScreen from "../screens/Home"
 import SettingsScreen from "../screens/Settings";
 import WorkoutDetails from "../screens/WorkoutDetails"
+import Discover from "../screens/Discover";
+
+const TabNavigator = createBottomTabNavigator(
+  { 
+    Home: {
+      screen:HomeScreen,
```

Comment:
Here you declare home to be a standalone screen instance, though we want to save the structure we already have. 
<img width="628" alt="TabNavigation" src="https://user-images.githubusercontent.com/8983928/59164145-40b95400-8ad7-11e9-8fe6-cb59d17fb3fa.png">

This is a visualization of what it was supposed to look like.
Here is an [example](https://stackoverflow.com/questions/52261716/is-a-stacknavigator-inside-tabnavigator-possible) of how to include navigation stacks to a tab navigation